text/0011-python-wrappers/README.md

```diff
@@ -0,0 +1,74 @@
+- Feature Name: ursa-python-wrappers
+- Start Date: 2018-12-19
+- RFC PR: 
+- URSA Issue: 
+
+# Summary
+[summary]: #summary
+
+A python wrapper for the URSA code base. 
+
+# Motivation
+[motivation]: #motivation
+
+Many projects in hyperledger and others use python and having a simple and easy to use wrapper
+will save many developers time and effort as well as making the wrapper uniform. 
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+
+The new directory structure
+
+```
+wrappers
```

Comment:
Why support 2 versions at all? It is a wrapper so it should contain minimal code. And even if 2 versions are needed, why 3.5 as most recent linux distros come with 3.6, why not 3.6 and 3.7

Reply:
The purpose for 3.5 is to support the Indy-Node/Plenum projects. The primary users of this wrapper. But from experience in indy-sdk I have often found that community members ask for 3.6 compatibilities.